witherspoon.xml

```diff
@@ -97448,6 +97456,58 @@
 		<default>NA</default>
 	</attribute>
 </targetInstance>
+<targetInstance>
+        <id>disable_tpm_sensor</id>
+        <type>unit-ipmi-sensor</type>
+        <is_root>false</is_root>
+        <instance_name>disable_tpm_sensor</instance_name>
+        <position>-1</position>
+        <attribute>
+                <id>BUS_TYPE</id>
+                <default>NA</default>
+        </attribute>
+        <attribute>
+                <id>CHIP_UNIT</id>
+                <default>0</default>
+        </attribute>
+        <attribute>
+                <id>CLASS</id>
+                <default>UNIT</default>
+        </attribute>
+        <attribute>
+                <id>IPMI_ENTITY_ID</id>
+                <default>0x04</default>
```

Comment:
Trying to ultimately get 0xCC04 as the IPMI sensor number.  The 0x04 comes from this value, the entity id, per this comment in attribute_types_hb.xml:

  <enumerationType>
    <id>SENSOR_NAME</id>
    <description>Enumeration indicating the IPMI sensor name, which will
        be used by hostboot when determining the sensor number to return.
        The sensor name consists of one byte of sensor type plus one byte of
        sub-type, to differentiate similar sensors under the same target.
        Our implementaion uses the IPMI defined entity ID as the sub-type.